project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.3.12
+sbt.version = 1.9.3
```

Comment:
What about bumping this to 1.9.9 (the current latest version)? Although this project is probably not affected, it seems there were a couple of security fixes in the last few releases.